src/docs/linea.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Linea
+subtitle: Linea is a ZK-Rollup powered by Consensys zkEVM. The protocol team centrally operates the sequencer. The team focuses on 100% EVM equivalence, open-sourcing the client, decentralizing the sequencer and allowing for multiple proving algorithms.
+labels:
+    - ZK Rollup
+    - EVM
+    - Mainnet
+links:
+    website:
+        url: https://linea.build/
+        name: linea.build
+    docs:
+        url: https://docs.linea.build/
+        name: Docs
+    l2beat:
+        url: https://l2beat.com/scaling/projects/linea
+        name: L2Beat
+    github:
+        url: https://github.com/OffchainLabs
+        name: Github
+    twitter:
+        url: https://twitter.com/LineaBuild
+        name: LineaBuild
+    discord:
+        url: https://discord.com/invite/linea
+        name: Discord
+---
+
+<Section title="Overview">
+
+Linea is a ZK-Rollup powered by Consensys zkEVM. The protocol team centrally operates the sequencer. The team focuses on 100% EVM equivalence, open-sourcing the client, decentralizing the sequencer and allowing for multiple proving algorithms.
+    ###### Focus
+
+    - 100% EVM equivalence
+    - Open source client node
+    - Censorship resistant withdrawals
+    - Diversified security council
+    - Decentralized operators and governance
+    - Multi-prover
+
+    <Labels labels={['ZK Rollup', 'EVM', 'Mainnet']} title="Linea" />
+
+</Section>
+
+<Section title="General">
+
+    <Parameter name="Block Time" value="12 seconds" tooltip="The rate at which the rollup produces blocks. Keep in mind that the value is subject to change in the future"/>
+
+    <Parameter name="Gas Limit" value="61 million" tooltip="The gas limit that can be consumed by an L2 block"/>
+
+    <Parameter name="Sequencing Frequency" value="30 - 120 seconds" tooltip="The frequency at which the rollup posts L2 transactions on Ethereum L1" />
+
+    <Parameter name="Finality" value="8 - 32 hours" tooltip="The time it takes for an L2 transaction to be considered final after it is sequenced on L1" />
+
+    <Parameter name="Rollup’s State Contract on L1" value={<Reference label="0xb32c3D0dDb0063FfB15E8a50b40cC62230D820B3" url="https://etherscan.io/address/0xb32c3D0dDb0063FfB15E8a50b40cC62230D820B3" />} tooltip="The contract used for sequencing, proving and storing the state of the L2 network" />
+
+    <Parameter name="State Trie" value="Sparse Merkle Trie + MiMC Hashing" tooltip="The data structure used to represent the state of the Rollup along with the hashing algorithm used to compute the root of the trie" />
+
+    <Parameter name="Node Implementations" value={[<Reference key="0" label="geth (Go)" url="https://github.com/Consensys/go-ethereum" />, <Reference key="1" label="linea-besu (Work in progress)" url="https://github.com/Consensys/linea-besu" />]} tooltip="The different client implementations of the Rollup" />
```

Comment:
This is not the node that is running the Linea rollup IMO. It is a simple geth fork without any changes.